### PR TITLE
Merge wallet compatibilities ideas and update docs

### PR DESCRIPTION
**Pull request by pilu**
I updated idea #142 since we are going to merge all the changes of #94 and #142 together.

I added more documentation, specified that the changes went through the security audit, and removed idea #94.

### CODE REVIEW

**Comment by oskarth** [comment]
Please resolve conflicts

File: ideas/142-wallet-compatibility/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+### Relevant reading
+
+* [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
+* [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
+* [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki)
+
+### User stories
+
+* As a user, I want to be able to use a `dapp` in Status, and then open the same `dapp` with the same account in a different Wall Software, using the 12 mnemonic words.
```
Comment (chadyj) [inline ideas/142-wallet-compatibility/README.md]: Wall > Wallet

File: ideas/142-wallet-compatibility/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## Product Overview
+
+Other wallets like MetaMask, Toshi, Cipher, etc., implements BIPs 39/32/44 like Status, and so they derive the keys starting from 12 mnemonic words.
+
+Given that they use the same standards, users can start playing with a `dapp` with one of those apps, and continue with another one without problems. They can open one of the other apps, and use the same 12 mnemonic words to import their existing account (private key).
+
+In Status, we implement the same standards to derive the keys starting from the 12 mnemonic words but using different constants, and we use the password added as an extra entropy in the generation of the seed.
+
+This means that users can click on "Add existing account" and successfully "import" an account with the 12 mnemonic words that they generated in a different wallet software, but ending up with a different key and address.
+Since the generation of the key is different, the account and address are not the ones that the user expected to import.
```
Comment (chadyj) [inline ideas/142-wallet-compatibility/README.md]: Can you briefly mention how to solve this? Is this a major problem?

File: ideas/142-wallet-compatibility/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Users won't be able to log in on `dapps` with the same they used previously, and won't be able to see the value and collectables they actually have in their wallet.
```
Comment (chadyj) [inline ideas/142-wallet-compatibility/README.md]: How do you intend to deal with this?

File: ideas/142-wallet-compatibility/README.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+* 20% of users send a transaction (OKR 2.4 of Q2)
+
+## Exit criteria
+
+Users can use a single account/address for `dapps` and use it both in external apps and in Status.
+
+## Supporting Role Communication
+
+Once deployed, the generation of the keys will change.
+It will be impossible to re-import an account created previously.
```
Comment (chadyj) [inline ideas/142-wallet-compatibility/README.md]: @j-zerah @pablanopete Please note!

File: ideas/142-wallet-compatibility/README.md (hunk 2 above)
Comment (pilu) [reply ideas/142-wallet-compatibility/README.md]: @chadyj in the Technical overview section, I describe the changes that fix it

**Pull request merged**